pkg/ingester/user_state.go

```diff
@@ -195,24 +199,36 @@ func (m *labelSetCounter) decreaseSeriesLabelSet(u *userTSDB, metric labels.Labe
 	}
 }
 
-func (m *labelSetCounter) UpdateMetric(u *userTSDB, vec *prometheus.GaugeVec) {
-	currentLbsLimitHash := map[uint64]struct{}{}
+func (m *labelSetCounter) UpdateMetric(ctx context.Context, u *userTSDB, vec *prometheus.GaugeVec) error {
```

Comment:
Is there a unit test covering this backfilling change?

Reply:
yeah... i added one!
https://github.com/cortexproject/cortex/pull/5955/files#diff-25e63d3b817863ee81ae61ea5bdf7dc4370cdc5033db9066d2afb1c415887d28R194-R208